redbot/core/core_commands.py

```diff
@@ -1877,6 +1877,52 @@ class Core(commands.Cog, CoreLogic):
         """Manage the bot's commands."""
         pass
 
+    @command_manager.group(name="listdisabled", invoke_without_command=True)
+    async def list_disabled(self, ctx: commands.Context):
+        """
+        List disabled commands.
+
+        If you're the bot owner, this will show global disabled commands by default.
+        """
+        # Select the scope based on the author's privileges
+        if await ctx.bot.is_owner(ctx.author):
+            await ctx.invoke(self.list_disabled_global)
+        else:
+            await ctx.invoke(self.list_disabled_guild)
+
+    @list_disabled.command(name="global")
+    async def list_disabled_global(self, ctx: commands.Context):
+        """List disabled commands globally."""
+        disabled_list = await self.bot._config.disabled_commands()
+        if disabled_list:
+            msg = _("{num:,} {cmd} {plur} disabled globally.\n").format(
+                num=len(disabled_list),
+                cmd=_("commands") if len(disabled_list) > 1 else _("command"),
+                plur=_("are") if len(disabled_list) > 1 else _("is"),
+            )
+            msg += box(", ".join(disabled_list))
+        else:
+            msg = _("There aren't any globally disabled commands.")
+        for page in pagify(msg):
```

Comment:
Splitting with pagify like this will break the code block if more than one message is needed. To use pagify in this way, you would need to apply the box to each `page`, not to the entire `msg`.